io.catenax.mp_standard_response/1.0.0/ProvideProductionForecast.ttl

```diff
@@ -0,0 +1,166 @@
+##########################################################################################
+# Copyright (c) 2023 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e. V.
+# Copyright (c) 2023 Siemens AG
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+##########################################################################################
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.mp_standard_response:1.0.0#> .
+@prefix ext-header: <urn:samm:io.catenax.messaging_header:1.0.0#> .
+@prefix ext-request: <urn:samm:io.catenax.mp_standard_request:1.0.0#> .
+@prefix ext-unsubscribe: <urn:samm:io.catenax.mp_standard_unsubscribe:1.0.0#> .
+
+:ProvideProductionForecast a samm:Aspect ;
+   samm:preferredName "ProvideProductionForecast"@en ;
+   samm:description "Answer to a customer with all information about the requested items"@en ;
+   samm:properties ( :productionForecastResponse :cxHeader ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:productionForecastResponse a samm:Property ;
+   samm:preferredName "productionForecastResponse"@en ;
+   samm:description "the concrete information about a production forecast"@en ;
+   samm:characteristic :ProductionForecastCharacteristic .
+
+:cxHeader a samm:Property ;
```

Comment:
You might consider importing the whole property from the shared messageHeader model